src/components/ebay-video/component.ts

```diff
@@ -111,6 +123,23 @@ class Video extends Marko.Component<Input, State> {
         }
     }
 
+    alignSeekbar() {
+        if (this.el) {
+            this.buttonPanel = this.el.querySelector<HTMLElement>(
+                ".shaka-controls-button-panel",
+            )!;
+            this.spacer = this.buttonPanel.querySelector(".shaka-spacer")!;
+            this.rangeContainer = this.el.querySelector<HTMLElement>(
+                ".shaka-range-container",
+            )!;
+        }
+        const buttonPanelRect = this.buttonPanel.getBoundingClientRect();
+        const spacerRect = this.spacer.getBoundingClientRect();
```

Comment:
Won't `this.buttonPanel` be undefined if the prior if statement failed? That would lead to a "document is undefined" error